api/spec/controllers/mno_enterprise/jpi/v1/organizations_controller_spec.rb

```diff
@@ -20,7 +20,7 @@ module MnoEnterprise
 
     # Stub organization + associations
     let(:metadata) { {} }
-    let!(:organization) { build(:organization, metadata: metadata, orga_invites: [], users: [], orga_relations: [], credit_card: credit_card) }
+    let!(:organization) { build(:organization, metadata: metadata, orga_invites: [], users: [], orga_relations: [], credit_card: credit_card, invoices: []) }
```

Comment:
@hedudelgado can you add a test scenario with rendering invoices so it exercises the view rendering?

Reply:
Hi @BrunoChauvet,
Done!